include/zephyr/bluetooth/conn.h

```diff
@@ -413,6 +413,124 @@ struct bt_conn_le_cs_fae_table {
 	uint8_t *remote_fae_table;
 };
 
+/** Channel sounding config main mode */
+enum bt_conn_le_cs_config_main_mode {
+	/** Mode-1 (RTT) */
+	BT_CONN_LE_CS_CONFIG_MAIN_MODE_1 = 1,
+	/** Mode-2 (PBR) */
+	BT_CONN_LE_CS_CONFIG_MAIN_MODE_2 = 2,
+	/** Mode-3 (RTT and PBR) */
+	BT_CONN_LE_CS_CONFIG_MAIN_MODE_3 = 3,
+};
+
+/** Channel sounding config sub mode */
+enum bt_conn_le_cs_config_sub_mode {
+	/** Unused */
+	BT_CONN_LE_CS_CONFIG_SUB_MODE_UNUSED = 0xFF,
+	/** Mode-1 (RTT) */
+	BT_CONN_LE_CS_CONFIG_SUB_MODE_1 = 1,
+	/** Mode-2 (PBR) */
+	BT_CONN_LE_CS_CONFIG_SUB_MODE_2 = 2,
+	/** Mode-3 (RTT and PBR) */
+	BT_CONN_LE_CS_CONFIG_SUB_MODE_3 = 3,
+};
+
+/** Channel sounding config role */
+enum bt_conn_le_cs_config_role {
```

Comment:
maybe we can rename this to

enum bt_conn_le_cs_role {
BT_CONN_LE_CS_ROLE_INITIATOR = BT_HCI_OP_LE_CS_INITIATOR_ROLE,
BT_CONN_LE_CS_ROLE_REFLECTOR = BT_HCI_OP_LE_CS_REFLECTOR_ROLE,
}

so that we can get rid of enum bt_cs_test_role

I say this assuming it's always 0 = initiator 1 = reflector throughout these commands

Reply:
Yes good shout